olmo/config.py

```diff
@@ -987,9 +987,10 @@ class TrainConfig(BaseConfig):
     How often to log to the console.
     """
 
-    gen1_gc_interval: int = 1
+    gen1_gc_interval: Optional[int] = 1
```

Comment:
Just checking, we want the default behavior to be messing with GC?

Reply:
Yes I think so. There might be a better default interval for smaller models, but this was a huge improvement for the 70B.